### ISSUE TITLE
WP_HOME / WP_SITEURL

### ISSUE THREAD

**Issue by dekode-andre** [DekodeInteraktiv/project-base#294]
In #174 / [application.php](https://github.com/DekodeInteraktiv/project-base/commit/7cef373034c74f8361b13ac2625ca4d78dd3b684#diff-53ea5a45a4b413ca49c7fac3f022e1dcbc2c203dcab4a1e7a34d1a8d65b26977) was changed from respecting `WP_HOME` / `WP_SITEURL` set in `.env` to be whatever `$http_host` is when the request comes in. Is there a reason for this, or can we please change it to respect `.env` again? There has been a couple of incidents already where this has caused confusion and effectively hurt a clients SEO as the site has been possible to use on an alternative URL.

**Comment by Clorith** [comment]
I agree we should do this, I can think of a few scenarios where the current implementation would make life easier during local environment setups, but in a distributed sense we should definitely not rely on the incoming request to determine canonical URL bases.

**Comment by Clorith** [comment]
Just had a check here, and this has since been corrected in 6786ae28499186179d2bb4b97203ada1ea9905bc, and the use of `$http_host` is now only used as fallback if there is no `WP_HOME` defined.

See https://github.com/DekodeInteraktiv/project-base/blob/51fd454b7ce3276c929399e3f65d4a1c9127aac2/config/application.php#L73-L85
